Units/parser-verilog.r/systemverilog-net-var.d/expected.tags

```diff
@@ -11,9 +11,10 @@ iBus	input.sv	/^  interconnect [0:1] iBus;$/;"	n	module:top
 rlMod	input.sv	/^module rlMod(input interconnect [0:1] iBus);$/;"	m
 iBus	input.sv	/^module rlMod(input interconnect [0:1] iBus);$/;"	p	module:rlMod
 Net_declarations	input.sv	/^module Net_declarations$/;"	m
+cap1	input.sv	/^  trireg (large) logic #(0,0,0) cap1;$/;"	n	module:Net_declarations
 addressT	input.sv	/^  typedef logic [31:0] addressT;$/;"	T	module:Net_declarations
-ecc	input.sv	/^  wire struct packed { logic ecc; logic [7:0] data; } memsig; \/\/ FIXME: ignored -> memsig$/;"	r	module:Net_declarations
-data	input.sv	/^  wire struct packed { logic ecc; logic [7:0] data; } memsig; \/\/ FIXME: ignored -> memsig$/;"	r	module:Net_declarations
```

Comment:
I don't know SystemVerilog. However, ecc and data look important tags for me.
Am I wrong?

Reply:
Good point, but this is OK.
SystemVerilog parser does not support struct/union members yet, as we discussed before on #2637.  It is still on my TODO list.

The struct members are tagged as a variable declaration (tag `r`) wrongly in this case.